package.json

```diff
@@ -29,28 +29,31 @@
     "ember-cli-version-checker": "^2.0.0"
   },
   "devDependencies": {
-    "ember-cli": "2.13.3",
-    "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.0.0",
-    "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.0",
-    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli": "~2.14.0",
+    "ember-cli-dependency-checker": "^2.0.1",
+    "ember-cli-eslint": "^4.1.0",
+    "ember-cli-htmlbars": "^2.0.2",
+    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
+    "ember-cli-inject-live-reload": "^1.6.1",
     "ember-cli-qunit": "^4.0.0",
     "ember-cli-shims": "^1.1.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^1.2.0",
```

Comment:
to be honest, I'm not a big fan of using `ember init` for CLI updates anyway... and adding unused dependencies just because CLI has them is not a good reason IMHO, even if they're only dev deps